docs/data-collection/scanning-notes.md

```diff
@@ -36,27 +36,28 @@
 
     ![adjustFOV.jpg](../assets/images/adjustFOV.jpg)
 
-## Eye-tracker calibration
-- [ ] Inform the participant about the calibration process (the [scanning protocol](scanning.md) provides you with scripts)
-- [ ] Run the experiment by pressing the green play button. 
-- [ ] Enter the session and participant number in the pop up window. The Eyelink system setup page opens.
-- [ ] Press <span class="keypress">C</span> to open the calibration mode to perform the calibration and validation.
-- [ ] Exit the calibration mode by clicking on  <span class="keypress">ESC</span> .
-- [ ] Once the calibration of the ET is concluded, hit the <span class="keypress">Esc</span> key on the laptop *{{ secrets.hosts.psychopy | default("███") }}* to exit the calibration mode continue the task's program.
-
-
-- [ ] Launch the ET calibration by pressing <span class="keypress">C</span> on the laptop keyboard or by clicking on Calibration on the ET interface:
-    - [ ] When the gaze is stable, the button `Accept fixation` appears green, then you can manually click on it to validate the first position.
+## Eye-Tracker Calibration Process
+
+- [ ] Brief the participant about the calibration process. Refer to the [scanning protocol](scanning.md) for provided scripts.
```

Comment:
```suggestion
```
The link should be the other way around - from the protocol to the place where calibration is described